src/styles/blocks/header.scss

```diff
@@ -0,0 +1,69 @@
+body,
+.header_nav a {
+  font-family: 'Roboto', sans-serif;
+}
+
+.header_top {
+  background-color: white;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 10px 20px;
+}
+
+.header_section {
+  justify-content: flex-end;
+  margin: 0 50px;
+}
+
+.header_logo {
+  display: flex;
+  align-items: center;
+  margin-left: 50px;
+  height: 40px;
+  width: 40px;
+}
+
+.header_nav {
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.header_nav a {
+  text-decoration: none;
+  color: inherit;
+  font-size: 12px;
+
+  font-size: 12px;
```

Comment:
The 'font-size: 12px;' property is declared twice for '.header_nav a'. Consider removing the duplicate declaration on line 39.